src/SFML/Graphics/Font.cpp

```diff
@@ -770,7 +771,7 @@ IntRect Font::findGlyphRect(Page& page, unsigned int width, unsigned int height)
         }
 
         // We can now create the new row
-        page.rows.push_back(Row(page.nextRow, rowHeight));
+        page.rows.emplace_back(Row(page.nextRow, rowHeight));
```

Comment:
`emplace_back()` can't be used in C++98, and is meant for cases where arguments are passed to the constructor. In this case you would just invoke the copy/move constructor.

`push_back()` has a `T&&` overload, so this would not be faster either.

Reply:
@Bromeon fix it.
https://github.com/SFML/SFML/pull/1732/commits/ae2d032d5a33e7f63fb2355b64426eef870eb0e5